<launch>
<!--Launch file for running the multirobot exploration-->

	<param name="robot_1/goal" value=""/>
	<param name="robot_2/goal" value=""/>
	<!-- <param name="robot_3/goal" value=""/> -->

	<param name="robot_1/get_goal_status" value="ready"/>
	<param name="robot_2/get_goal_status" value="ready"/>

	<param name="robot_1/need_change_radius" value="no"/>
	<param name="robot_2/need_change_radius" value="no"/>


	<!-- <group ns="/robot_1" >
		<include file="$(find road_based_explorer)/launch/include/start_detector.launch">
			<arg name="robot_name" value="robot_1"/>
		</include>

		<include file="$(find road_based_explorer)/launch/include/start_filter.launch">
			<arg name="robot_name" value="robot_1"/>
			<arg name="exploation_points" value="[(-20, 5), (60, -55)]"/>
		</include>

		<include file="$(find road_based_explorer)/launch/include/start_assigner.launch">
			<arg name="robot_name" value="robot_1"/>
		</include>
	</group> -->

	<group ns="/robot_2" >
		<include file="$(find road_based_explorer)/launch/include/start_detector.launch">
			<arg name="robot_name" value="robot_2"/>
		</include>

		<include file="$(find road_based_explorer)/launch/include/start_filter.launch">
			<arg name="robot_name" value="robot_2"/>
			<arg name="exploation_points" value="[(-40, 5), (20, -55)]"/>
		</include>

		<include file="$(find road_based_explorer)/launch/include/start_assigner.launch">
			<arg name="robot_name" value="robot_2"/>
		</include>
	</group>

	<!-- run RViz node (visualization) -->
	<!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find hunter2_explorer)/launch/include/rviz_config/hunter_1.rviz">
		<remap from="move_base_simple/goal" to="robot_1/move_base_simple/goal"/>
		<remap from="clicked_point" to="robot_1/clicked_point"/>
	</node> -->

</launch>